truss/contexts/image_builder/serving_image_builder.py

```diff
@@ -82,6 +83,28 @@ def copy_into_build_dir(from_path: Path, path_in_build_dir: str):
         # Download external data
         download_external_data(self._spec.external_data, data_dir)
 
+        # Download from HuggingFace
+        model_files = {}
+        if config.hf_cache:
+            (build_dir / "cache_warmer.py").write_text(
```

Comment:
nit: let's move this out into its own file. No need to keep this script in a string